phylogeny/prototypeSelection.py

```diff
@@ -125,3 +125,82 @@ def prototype_selection_exhaustive(dm: DistanceMatrix, num_prototypes: int,
         if d > max_dist:
             max_dist, max_set = d, s
     return max_set
+
+
+def prototype_selection_constructive_maxdist(dm, num_prototypes, steps=10):
+    '''Heuristically select k prototypes for given distance matrix.
```

Comment:
Didn't know that. Thanks for pointing me to it. I rearranged my documentation to adhere to this convention. Let me know if it suits now.